pydov/search/abstract.py

```diff
@@ -117,8 +117,28 @@ def _init_wfs(self):
         to all subclasses and instances.
         """
         if AbstractSearch.__wfs is None:
-            AbstractSearch.__wfs = WebFeatureService(
-                url=build_dov_url('geoserver/wfs'), version="1.1.0")
+            capabilities = None
+
+            for hook in pydov.hooks:
+                capa = hook.inject_meta_response(
+                    build_dov_url('geoserver/wfs') + '?version=1.1.0')
```

Comment:
just a question: if you hard-code version 1.1.0, this will need to be adjusted when changed to 2.0?

Reply:
Yes, this is one of the things that would need to be changed when we upgrade.